src/inquirer/render/console/_checkbox.py

```diff
@@ -89,6 +91,11 @@ def process_input(self, pressed):
         elif pressed == key.RIGHT:
             if self.current not in self.selection:
                 self.selection.append(self.current)
+        elif pressed == key.CTRL_A:
+            for x in self.question.choices:
+                self.selection = list(range(len(self.question.choices)))
+        elif pressed == key.CTRL_Q:
+            self.selection = []
         elif pressed == key.ENTER:
```

Comment:
Also I see there are no mentions of the changes on documentation and change-logs.